unlock-app/src/middlewares/postMessageMiddleware.js

```diff
@@ -0,0 +1,26 @@
+import { OPEN_MODAL_IN_NEW_WINDOW } from '../actions/modal'
+
+// cribbed from https://stackoverflow.com/questions/326069/how-to-identify-if-a-webpage-is-being-loaded-inside-an-iframe-or-directly-into-t
+export function inIframe(window) {
```

Comment:
Probably worth putting in `config` (along with `isServer` and co) just in case we want to re-use it?

Reply:
I see what you mean. Perhaps "config" is a misnomer, it's more like we're setting up environment. Food for thought in a later PR. I'll slide this over there. One question is how I should access it then? import just that function for this middleware? In any case, this seems to be a separate PR since it will be doing 2 things (adding config and making a new middleware) so I'll extract that out first

Reply:
#1350 implements what you're describing, I'll rebase when that's merged

Reply:
;) Great! Thanks, good point about the name too!